i18n/en.json

```diff
@@ -64,9 +64,13 @@
 	"error-wikis": "$1 {{PLURAL:$1|wiki|wikis}} {{PLURAL:$1|is|are}} invalid. Check the spelling.",
 	"event": "Event",
 	"event-all-edits": "All edits",
+	"event-all-edits-list": "All edits list",
```

Comment:
I talked to Joe and he said we can simply use the existing "All edits" message (`event-all-edits`). The ticket has been updated accordingly

Reply:
Oh cool. Fixed.